pkg/controllers/consolidation/pdblimits.go

```diff
@@ -0,0 +1,86 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consolidation
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/policy/v1beta1"
```

Comment:
Use policy/v1? v1beta1 has been removed from k/k now.

Reply:
Thanks, updated.

Reply:
I need to revert this change, we currently test against 1.19, 1.20, 1.21 and 1.22 and this breaks on older K8s.